core/server/src/main/java/alluxio/worker/block/ClientRWLock.java

```diff
@@ -29,7 +29,7 @@
 public final class ClientRWLock implements ReadWriteLock {
   // TODO(bin): Make this const a configurable.
   /** Total number of permits. This value decides the max number of concurrent readers. */
-  private static final int MAX_AVAILABLE = 100;
+  private static final int MAX_AVAILABLE = 1000;
```

Comment:
this value decides the max number of concurrent reader clients for each individual block. I don't even think 100 will be achieved right now in reality (i.e., 100 clients talking to one worker on one single block). So I guess it might not matter if it is 100 or 1000.